dev/README_RELEASE_AIRFLOW.md

```diff
@@ -148,12 +148,10 @@ svn commit -m "Add artifacts for Airflow ${VERSION}"
 At this point we have the artefact that we vote on, but as a convenience to developers we also want to
 publish "snapshots" of the RC builds to pypi for installing via pip. To do this we need to
 
-- Edit the `setup.py` to include the RC suffix.
-
 - Build the package:
 
     ```shell script
-    python setup.py compile_assets sdist bdist_wheel
+    python setup.py compile_assets egg_info --tag-build "$(sed -e "s/^[0-9.]*//" <<<"$VERSION")" sdist bdist_wheel
```

Comment:
does `egg_info` take care of both `sdist bdist_wheel` ?

I guess yes but just confirming

Reply:
Both of those use egg_info (technically they use build which uses egg_info)

But yes, I used this for https://pypi.org/project/apache-airflow-upgrade-check/1.1.0rc1/#files

Reply:
Nice. I might use it in the providers packages as well and get rid of the setup.py "if" I have there.